common/src/stack/command/stack/commands/config/host/interface/__init__.py

```diff
@@ -48,11 +49,12 @@ class Command(stack.commands.config.host.command):
 	"""
 
 	def run(self, params, args):
-		(interface, mac, module, flag) = self.fillParams([
+		(interface, mac, module, flag, no_sync_config) = self.fillParams([
```

Comment:
I would change the new parameter to just be `sync` and default it to True. Other commands with this same behavior use either `sync` or `syncnow` but I think `sync` is the best choice.

Also, make sure to add this new parameter to the XML docs above. Those docs are used to generate the help string.

Also, the common pattern is to add a line converting the parameter to a boolean right afer the `fillParams` call:

```
sync = self.str2bool(sync)
```